README.md

```diff
@@ -1,148 +1,180 @@
 # Linux steering wheels FFB support
 
-
 ## Ranks
-Ranks are totally subjective.
 
-### Native
-Device works perfectly out-of-the-box, no need to build/install any custom drivers.
```

Comment:
Why do you got rid of the **native** rank? 
My thought process was: there is custom drivers and pidff one. If a device initializes and works without custom drivers, that's **native**. If there is some things like "0xa7 descriptor missing", then it's not native until pidff in upstream changes it, for example.

But i like how the table looks without it :)

Reply:
It's not really a rank, "native" doesn't say how well a driver works. I guess it means that the driver is included in the kernel. And that indicates a good thing, the user won't have to install drivers, but it misses some information.

For example, hid-logitech is native but I would give it only a Bronze because it lacks many effects. It can be Platinum with the new-lg4ff module but then it's not native.

I think it was confusing as a rank level. I though about adding it as a column if the driver was inlined in the kernel, but I though that information could better be explained in footnotes or sections dedicated to each driver.

Also, the term native usually means something different, in contrast to emulated.

Reply:
Aha, okay, i understand now. I wanted something what protondb does. Native means 'native' like 'compiled for Linux', and everything else (even Platinum) working through Proton

Reply:
> Aha, okay, i understand now. I wanted something what protondb does. Native means 'native' like 'compiled for Linux', and everything else (even Platinum) working through Proton

Oh, but are there wheels only working with Proton? If that's the case then we would need a column.

Anyway, that's not how it was used. Logitech wheels were Platinum but they don't need Proton. Thus I thought it meant something different.

Reply:
Not really.
As you said, logitech wheels are 'native' but not good, because not every effect implemented